src/assignments/candidateQuiz.rst

```diff
@@ -0,0 +1,180 @@
+testable parts. The goal is to get simple parts working first and then expand
+the code in a systematic way. The following is NOT the only way to complete
+this assignment, but it provides a framework for thinking through the project.
+
+Create a replit.com project to start building your project.  Make sure "Candidate Quiz" is part of the name.
+Use the textbook for directions.  
+
+Part 1: Minimum Viable Quiz
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Define variables for:
+
+   a. candidate's name
+   b. a quiz question (pick any question from the table in Part 2 below)
+   c. the correct answer
+   d. the candidate's response
+
+#. Ask for the candidate's name. Before moving to the next step, use
+   ``Console.WriteLine`` to verify that your code correctly stores the information.
+#. Display the question and prompt the candidate for an answer. As before, use
+   ``Console.WriteLine`` to verify that your program correctly stored the answer.
+#. Check the candidate's answer to see if it is correct.
+#. Provide basic feedback to the candidate. This should include their name and
+   whether their answer was correct.
+
+.. admonition:: Note
+
+   If not already done, remove the extra ``Console.WriteLine`` statements from steps 2 & 3. Make sure your small app works properly before moving on to part 2.
+
+Part 2: Use Arrays
+^^^^^^^^^^^^^^^^^^
+
+Now that your small app is working, expand it to deal with multiple questions.
+
+#. Redefine your question and correct answer variables to be arrays.
+#. Fill these arrays with the questions and answers listed in the table below.
+#. You still need to ask for the candidate's name.
+#. Using bracket notation, select one question and use that to prompt the
+   candidate.
+#. Compare the candidate's response to the proper entry in the answers array.
+#. Replace the basic feedback with string interpolation.
+
+.. admonition:: Note
+
+   Checking for the correct answer should be case insensitive (e.g. "Orbit" is the same as "orbit" or "ORBIT").
+
+.. list-table::
+   :header-rows: 1
+
+   * - Question
+     - Answer
+
+   * - True or false: 5000 meters = 5 kilometers.
+     - "True"
+
+   * - (5 + 3)/2 * 10 = ?
+     - "40"
+
+   * - Given the array ``[8, "Orbit", "Trajectory", 45]``, what entry is at index 2?
+     - "Trajectory"
+
+   * - Who was the first American woman in space?
+     - "Sally Ride"
+
+   * - What is the minimum crew size for the International Space Station (ISS)?
+     - "3"
+
+Part 3: Use Iteration to Ask All Questions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Add one or more loops to your code to ask all the questions in the quiz.
+Use arrays to collect and check all the candidate's answers.  Finally,
+calculate the candidate's score and print the results.
+
+Helpful hint - To calculate the candidate's percentage, use the equation:
+
+   (Number of Correct Answers) / (Number of Questions) * 100
+
+Note that the final report MUST have the format shown in the "Results Output"
```

Comment:
technically, this is called "Example Output" here. and I'm not convinced that it *must* have the same format. Does it? Could we just say, it should look like the example output?

Reply:
works for me
that was pulled over from the JS book and they used a template literal.   I think this was the hint for that.